package.json

```diff
@@ -1,6 +1,7 @@
 {
   "name": "@doordeck/javascript-doordeck-sdk",
   "title": "Javascript Doordeck Sdk",
+  "type": "module",
```

Comment:
needed?

Reply:
ah I see the `node --experimental-vm-modules`

Reply:
This was needed to be able to run the tests. But, if it makes an issue, we can keep some changes local